CONTRIBUTING.md

```diff
@@ -337,6 +337,9 @@ Some further tips for good type hints:
   possible type, and `Text` if it can be either `unicode` or `bytes`;
 * use platform checks like `if sys.platform == 'win32'` to denote
   platform-dependent APIs.
+* `__enter__` methods and other methods that return instances of the
```

Comment:
This doesn't seem like a good place for this. There's `__exit__` stuff below. Maybe have `__enter__` and `__exit__` stuff together?

Reply:
Thanks, I missed that. Another argument for reorganizing this section (see my other comment) :)